python/sglang/srt/managers/router/radix_cache.py

```diff
@@ -38,6 +38,13 @@ def __init__(self, disable=False):
         self.disable = disable
 
     ##### Public API #####
+
+    def clear(self):
+        self.root_node = TreeNode()
+        self.root_node.value = []
+        self.root_node.ref_counter = 1
+        self.evictable_size_ = 0
```

Comment:
nit: You can name this function `reset` so that you can reuse this function like

```
def __init__(self, disable=False):
    self.disable = disable
    self.reset()
```
Also should we call `torch.cuda.empty_cache()` to avoid fragmentation?